### PR TITLE
Bootstrap rebased

### PR DESCRIPTION
**Pull request by jboucher154**
This is a PR for the application of basic bootstrap styling of the project (not including the gameplay screen). **note:  the work for this PR was done on PC-20-bootstrap, this branch was made to properly rebased and create the PR**

- New fonts were added and can be found in the styles.css file
- forms were standardized which included modifying the Django form classes
- The navbar was moved from the index to be rendered with the various templates from the backend

### CODE REVIEW

File: srcs/backend/transcendence/srcs/app/play.py
Comment (tikanoja) [inline srcs/backend/transcendence/srcs/app/play.py:1]: lets not delete these, uncommit / accept the develop version of this file?

File: srcs/backend/transcendence/srcs/app/user/account.py
Comment (tikanoja) [inline srcs/backend/transcendence/srcs/app/user/account.py:1]: lets not delete these, uncommit / accept the develop version of this file?

File: srcs/backend/transcendence/srcs/pong/utils.py
Comment (tikanoja) [inline srcs/backend/transcendence/srcs/pong/utils.py:1]: nice formatting with data_to_send but I would love to keep the logger print on 92 for lil bit

File: srcs/backend/transcendence/srcs/app/play.py
Comment (jboucher154) [reply srcs/backend/transcendence/srcs/app/play.py:1]: done, I checked out current develop and saved that version

File: srcs/backend/transcendence/srcs/app/user/account.py
Comment (jboucher154) [reply srcs/backend/transcendence/srcs/app/user/account.py:1]: done, I checked out current develop and saved that version

File: srcs/backend/transcendence/srcs/pong/utils.py
Comment (jboucher154) [reply srcs/backend/transcendence/srcs/pong/utils.py:1]: I put the print back in

**Review by tikanoja — APPROVED** [review]
Wonderful stuff! Checked and looks good to me!

**Pull request merged**